### ISSUE TITLE
npm init stencil does not work behind corporate proxy

### ISSUE THREAD

**Issue by Ritzlgrmft** [ionic-team/create-stencil#16]
`create-stencil` is using the node https module for downloading the template. This module seems to ignore all proxy settings I have set:

- `PROXY` environment variable
- npm proxy setting
- git proxy setting

I always see (independent from the template):

```text
>npm init stencil
npx: installed 1 in 4.944s

√ Pick a starter » ionic-pwa
√ Project name » stencil-pwa

? Preparing starterevents.js:183
      throw er; // Unhandled 'error' event
      ^

Error: connect ETIMEDOUT 140.82.118.4:443
    at Object._errnoException (util.js:1022:11)
    at _exceptionWithHostPort (util.js:1044:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
```

**Comment by walpolea** [comment]
Have the same issue

**Comment by dasois** [comment]
any news on how to get this work with stencil init?

**Comment by BrunoRodolpho** [comment]
same here:

npm init stencil app test
npx: installed 1 in 33.415s
⠙ Preparing starterevents.js:173
      throw er; // Unhandled 'error' event
      ^

Error: getaddrinfo ENOTFOUND github.com
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:60:26)
Emitted 'error' event at:
    at TLSSocket.socketErrorListener (_http_client.js:409:9)
    at TLSSocket.emit (events.js:196:13)
    at emitErrorNT (internal/streams/destroy.js:91:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:59:3)
    at processTicksAndRejections (internal/process/task_queues.js:84:9)

**Comment by rgunczer** [comment]
Same here:
✖ Cannot download "https://github.com/ionic-team/stencil-component-starter/archive/master.zip
  Check your internet connection

  Error: getaddrinfo ENOTFOUND github.com github.com:443"

Is there any workaround? Maybe something similar what Cypress.io has? Specify zip location etc.

**Comment by dasois** [comment]
There is a pull request for this, pls merge!

**Comment by Manu1909** [comment]
Any update on this?

**Comment by sajTempler** [comment]
seems merged in #31

**Comment by manucorporat** [comment]
Fixed! https://github.com/ionic-team/create-stencil/commit/bbb855ab32018221e926355181a3d68fe0db5481
